misc/help.talon

```diff
@@ -1,4 +1,18 @@
 help alphabet: user.help_alphabet(user.get_alphabet())
+help symbols: user.help_symbol_key_words(user.get_symbol_key_words())
```

Comment:
For me help symbols overflows the bottom of the screen. Maybe you could do some kind of pagination stuff like 'help context'. If not, you might be able to improve the situation by merging synonyms together, e.g. on my setup 'angle', 'left angle', and 'less than' all map to '<' so that could be formatted '<: angle, left angle, less than' and this would make the window wider so that more could fit on the screen at a time.